common/entity/Lecture.ts

```diff
@@ -92,4 +92,10 @@ export class Lecture {
         array: true,
     })
     declinedBy: User['userID'][];
+
+    @Column({ default: null, nullable: true })
+    zoomMeetingId: string;
+
+    @Column({ nullable: true, type: 'json' })
+    zoomMeetingReport: any | null;
```

Comment:
How does the meeting report look like? Maybe we could extract participant information (i.e. a table (appointmentId, userId, joinedAt, leftAt, duration)) to make it easier to work with this information afterwards

Reply:
The response is a json object and looks like this (example provided by zoom)
:

```
{
    "next_page_token": "Tva2CuIdTgsv8wAnhyAdU3m06Y2HuLQtlh3",
    "page_count": 1,
    "page_size": 30,
    "total_records": 1,
    "participants": [
        {
            "customer_key": "349589LkJyeW",
            "duration": 259,
            "failover": false,
            "id": "30R7kT7bTIKSNUFEuH_Qlg",
            "join_time": "2022-03-23T06:58:09Z",
            "leave_time": "2022-03-23T07:02:28Z",
            "name": "example",
            "registrant_id": "example",
            "user_email": "jchill@example.com",
            "user_id": "27423744",
            "status": "in_meeting",
            "bo_mtg_id": "27423744",
            "participant_user_id": "DYHrdpjrS3uaOf7dPkkg8w"
        }
    ]
}
```

Reply:
this is tracked in https://github.com/corona-school/project-user/issues/738